### PR TITLE
Metrics beta end - disclaimer updates

### PR DESCRIPTION
**Pull request by ale-cota**
## DESCRIBE YOUR PR
Updates to communicate Metrics beta end and retiring product. Should not merge until external comms are done on September 4th.

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [x] Other deadline: September 4th
- [ ] None: Not urgent, can wait up to 1 week+

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

### CODE REVIEW

**Review by ArthurKnaus — APPROVED** [review]

**Review by matejminar — APPROVED** [review]

File: docs/platforms/android/metrics/index.mdx [hunk 1]
```diff
@@ -5,8 +5,6 @@ sidebar_order: 5500
 ---
```
Comment (stephanie-anderson) [inline docs/platforms/android/metrics/index.mdx:5]: I would add the following to the pages' frontmatter, it removes the page from the sidebar, but makes it still viewable with direct link. This way we can avoid new users finding that page, and being confronted with the fact it's no longer meaningful for them.

```suggestion
sidebar_hidden: true
---
```

**Review by vivianyentran — APPROVED** [review]
These changes look good to me! I like Stephanie's suggestion to hide these pages in the sidebar.

I also did a quick search for references to "/product/explore/metrics" and found a few more things to remove (can be in a separate PR):

- Remove [metrics section](https://docs.sentry.io/product/#metrics) from Product Walkthroughs page
- Remove custom metrics note in [Performance Metrics page](https://docs.sentry.io/product/performance/metrics/)
- Remove mention of metrics in [Dashboards page](https://docs.sentry.io/product/dashboards/#open-dashboard-widgets-in-discover-issues-and-metrics)
- Remove the last bullet about metric tags in [Metric Alerts Configuration - Tags & Properties](https://docs.sentry.io/product/alerts/create-alerts/metric-alert-config/#tags--properties)

LMK if you need help with these changes!

**Comment by ale-cota** [comment]
hi, @vivianyentran!
Thanks a lot for the review and the suggestions. If you could help with implementing those in this or a separate PR that would be amazing, very much appreciated 🙏

**Comment by vivianyentran** [comment]
Created a separate PR #11231 for the additional doc updates

**Pull request merged**